src/client/core.ts

```diff
@@ -81,11 +78,16 @@ export async function aptosRequest<Req extends {}, Res extends {}>(
     url: fullUrl,
   };
 
+  // Handle case for `Unauthorized` error (i.e API_KEY error)
+  if (result.status === 401) {
```

Comment:
May want to include 403 in here as well later.

Reply:
interesting... what is 403 status for?

Reply:
It's forbidden, rather than 401 unauthorized

It's honestly used interchangeably sometimes.  But, if 401 is the only one that's thrown right now we can stick with it

Reply:
ohh yea, server throws 401